server/datastore/mysql/migrations_test.go

```diff
@@ -70,6 +70,8 @@ func createMySQLDSForMigrationTests(t *testing.T, dbName string) *Datastore {
 }
 
 func Test20210819131107_AddCascadeToHostSoftware(t *testing.T) {
+	t.Skip("doesn't work since code is using the latest schema and the schema here is missing the bundle_identifier")
```

Comment:
Should we just delete this test then? I wonder if there's a good solution for automated testing of migrations like this.